ui/common/css/header/_title.scss

```diff
@@ -8,21 +8,21 @@
   margin: 0 0.5rem;
   display: none;
 
-  @include breakpoint($mq-xx-small) {
+  @media (min-width: at-least($xx-small)) {
     display: block;
   }
 
-  @include breakpoint($mq-topnav-hidden) {
+  @media (max-width: at-most($medium)) {
     /* move away from hamburger */
     margin-#{$start-direction}: calc(0.5rem + #{$site-header-height});
   }
 
-  @include breakpoint($mq-site-header-tall) {
+  @include mq-site-header-tall {
     font-size: 30px;
     line-height: #{$site-header-tall-height - 3};
   }
 
-  @include breakpoint($mq-topnav-visible $mq-site-header-tall) {
+  @media (min-width: at-least($medium)) and (min-height: at-least($tall)) {
```

Comment:
here I suppose is where the mixin would fall short?

Reply:
you could make mq-topnav-visible-and-tall but the media query is good here too. when there's only a handful of occurrences of a given query, i get on the fence about whether to give them a dedicated mixin name. probably should have for mq-subnav-top above

Reply:
ok, nice. Yeah I think in general we should be generous with these mixins.